frame/session/src/lib.rs

```diff
@@ -693,7 +693,6 @@ impl<T: Config> Pallet<T> {
 				if let Some(&(_, ref old_keys)) = now_session_keys.next() {
 					if old_keys != keys {
 						changed = true;
-						return
```

Comment:
Doesn't dropping the return here change behavior?

Reply:
No, note that it's in a lambda function and the return exits that function not the `rotate_session` function. (It would have been the last instruction in the function anyway),

Reply:
ah yes so it is. My bad.